recipe/meta.yaml

```diff
@@ -1,61 +1,284 @@
+    folder: externals/nasm-2.11.06                                                   # [win]
+    sha256: de3c87b26a80e789986d8e6950c6304175d3829afe9c6c7211eb7257266ab0ac         # [win]
+  - url: https://github.com/python/cpython-bin-deps/archive/libffi.zip               # [win]
+    folder: externals/libffi                                                         # [win]
+    sha256: 4872e72e188a5aa1124db0c3b163a4163e84ead359a514d86dd7c6fa2d2ff02a         # [win]
 
 build:
-  number: 0
-  # Windows has issues updating python if conda is using files itself.
-  # Copy rather than link.
-  no_link:
-    - DLLs/_ctypes.py   # [win]
-    - bin/python3.6     # [unix]
-  track_features:
-    - vc14              # [win]
-
-requirements:
-  build:
-    - bzip2 1.0.*       # [unix]
-    - openssl 1.0.*     # [unix]
-    - readline 6.2*     # [unix]
-    - sqlite 3.13.*     # [unix]
-    - tk 8.5.*          # [unix]
-    - xz 5.2.*          # [unix]
-    - zlib 1.2.*        # [unix]
-    - ncurses 5.9*      # [linux]
-  run:
-    - openssl 1.0.*     # [unix]
-    - readline 6.2*     # [unix]
-    - sqlite 3.13.*     # [unix]
-    - tk 8.5.*          # [unix]
-    - xz 5.2.*          # [unix]
-    - zlib 1.2.*        # [unix]
-    - ncurses 5.9*      # [linux]
-    - vs2015_runtime    # [win]
-
-test:
-  commands:
-    - python -V
-    - python3 -V            # [unix]
-    - 2to3 -h
-    - pydoc -h
-    - python3-config --help  # [unix]
+  number: 8
+  script_env:
+    - PY_INTERP_LINKAGE_NATURE
+    - PY_INTERP_DEBUG
+
+outputs:
+  - name: python
+    build:
+      activate_in_script: True
+      # Windows has issues updating python if conda is using files itself.
+      # Copy rather than link.
+      no_link:
+        - DLLs/_ctypes.pyd  # [win]
+      # We delete the shared libraries.
+      ignore_run_exports:   # [unix]
+        - bzip2             # [unix]
+      # Disabled until new conda-build release
+      detect_binary_files_with_prefix: True
+      # detect_binary_files_with_prefix: False
+      # binary_has_prefix_files:
+      #   - lib/libpython3.8.*.1.0
+      #   - bin/python3.8  # [linux]
+      #   - lib/libpython3.8.a  # [linux]
+      #   - lib/libpython3.8.nolto.a  # [linux]
+      #   - lib/libpython3.so  # [linux]
+      #   - lib/python3.8/lib-dynload/_hashlib.cpython-38-x86_64-linux-gnu.so  # [linux]
+      #   - lib/libpython3.dylib  # [osx]
+      # match python.org compiler standard
+      skip: True            # [win and int(float(vc)) < 14]
+      skip_compile_pyc:
+        - '*.py'            # [osx and arm64]
+      # Would like to be able to append to the calculated build string here, ping @msarahan, is this possible?
+      string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}{{ linkage_nature }}{{ debug }}_cpython
+      run_exports:
+        noarch:
+          - python
+        weak:
+          - python_abi 3.8.* *_cp38
+    # Not until: https://github.com/mingwandroid/conda-build/commit/c68a7d100866df7a3e9c0e3177fc7ef0ff76def9
```

Comment:
@mingwandroid is this upstream in conda-build? Can we uncomment the lines below?

Reply:
Yes it is. Setting this variable should work ok. I need to add a check to bld.bat for defaults but I'll do that when I recsync.

Reply:
Thanks!